media/js/base/mozilla-modal.js

```diff
@@ -78,10 +78,8 @@ Mozilla.Modal = (function(w, $) {
             }
         });
 
-        $modal.hide();
-        $modal.fadeIn('fast', function() {
-            $modal.focus();
-        });
+        $modal.show();
```

Comment:
Hm, I removed it (seemingly) without consequence. I think the only place the `#modal` element is ever set to `display: none` is on the (removed) line 81.